Modules/Devices/test/Devices.Domain.Tests/Tiers/TierIdTests.cs

```diff
@@ -61,7 +63,7 @@ public void Cannot_create_tier_id_with_invalid_id_characters()
     {
         char[] invalidCharacters = { '|', '-', '!' };
         var validTierIdPrefix = "TIR";
-        var tierIdValue = TestDataGenerator.GenerateString(TierId.DEFAULT_MAX_LENGTH_WITHOUT_PREFIX, invalidCharacters);
+        var tierIdValue = TestDataGenerator.GenerateString(StronglyTypedId.DEFAULT_MAX_LENGTH_WITHOUT_PREFIX, invalidCharacters);
```

Comment:
```suggestion
        var TIER_ID_WITH_INVALID_CHARACTER = "TIRxxxxxxxxxxxxxxxx!";
```

You can also remove the `invalidCharacters` variable. It doesn't make sense to test more than one. As long as you don't test every single invalid character, testing more than one doesn't improve the chance that there isn't one invalid character that ISN'T recognized.